content/200-concepts/100-components/04-introspection.mdx

```diff
@@ -166,6 +139,12 @@ model User {
 }
 ```
 
+Trying to generate your Prisma Client with `prisma generate` you would get the following error:
+
+```
+TODO Add validation error message from `generate`
```

Comment:
```suggestion
$ npx prisma generate

Error: Schema parsing
error: The model "User" cannot be defined because a model with that name already exists.
  -->  schema.prisma:17
   |
16 | }
17 | model User {
   |

Validation Error Count: 1
```